Software/real_time/ROS_RoboBuggy/src/robobuggy/scripts/transistor_sweep_test.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+import rospy
+from robobuggy.msg import Command
+
+def sweep_steering():
+    pub = rospy.Publisher('Command', Command, queue_size=1000)
+    rospy.init_node('steering_sweep', anonymous=True)
+    rate = rospy.Rate(20) # 1Hz
```

Comment:
I thought this means 20Hz? Or did you just not update the comment?

Reply:
Yeah forgot to update the comment. Started at 1Hz and upped it to 20Hz to get a cleaner sweep of the steering.